OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -307,7 +307,8 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
         listView.setOnItemLongClickListener(new AdapterView.OnItemLongClickListener() {
             @Override
             public boolean onItemLongClick(AdapterView<?> parent, View view, int position, long id) {
-                if (((IconImageView) view.findViewById(R.id.bulk_download)).getIcon() == FontAwesomeIcons.fa_check) {
+                IconImageView bulkDownload = (IconImageView) view.findViewById(R.id.bulk_download);
```

Comment:
A couple of nits:
- `bulkDownload` should be final.
- rename `bulkDownload` to `bulkDownloadIcon`